numpyro/svi.py

```diff
@@ -12,7 +14,34 @@ def _seed(model, guide, rng):
 
 
 def svi(model, guide, loss, optim_init, optim_update, get_params, **kwargs):
+    """
+    Stochastic Variational Inference given an ELBo loss objective.
+
+    :param model: Python callable with Pyro primitives for the model.
+    :param guide: Python callable with Pyro primitives for the guide
+        (recognition network).
+    :param loss: ELBo loss, i.e. negative Evidence Lower Bound, to minimize.
+    :param optim_init: initialization function returned by a JAX optimizer.
+        see: :mod:`jax.experimental.optimizers`.
+    :param optim_update: update function for the optimizer
+    :param get_params: function to get current parameters values given the
+        optimizer state.
+    :param kwargs: static arguments for the model / guide, i.e. arguments
```

Comment:
nit: missing `dict` for type

Reply:
kwargs here is just a loose collection of python `**kwargs`, will change to `**kwargs` for clarity.